news/amd.rst

```diff
@@ -0,0 +1,25 @@
+**Added:**
+
+* Functions to compute amd and pdd in src/diffpy/metrics/amd.py
+* Tests for amd and pdd functions in tests/test_amd.py
+* Test data (two cif files and pkl files (DataFrames)) in tests/test_data
+
+**Changed:**
+
+* Make pre-commit ignore test_data/ folder (in .pre-commit-config.yaml)
```

Comment:
this also not needed.  What is put in news is user-facing things that will appear in the changelog.   Also, a "change" refers to functionality that is changed.   We don't need this statement, but if we did it would fit better under "fixed".  No problem, just letting you know for the the future.